docs/fiddles/features/keyboard-shortcuts/global/index.html

```diff
@@ -7,10 +7,5 @@
 </head>
 <body>
     <h1>Hello World!</h1>
-    <p>
```

Comment:
Nit: We could describe which global shortcut we want to use here!

Reply:
Added a description to each fiddle!